server/router/dispatcher.ts

```diff
@@ -45,6 +45,9 @@ router.post('/', (req, res) => {
 });
 
 // Remove Dispatcher
-router.delete('/:id', (req, res) => deleteByID(req, res, docClient, 'Dispatchers'));
+router.delete('/:id', (req, res) => {
+  const { id } = req.params;
+  deleteByIDOld(res, docClient, id, 'Dispatchers');
```

Comment:
Why use this old version of the function when you can use the new version?

Reply:
at the time i wasnt sure if the diff would be too big if i added a new model for dispatchers, but i guess it wouldnt be too different